package.json

```diff
@@ -243,6 +243,10 @@
         "command": "mdb.exportToRuby",
         "title": "MongoDB: Export To Ruby"
       },
+      {
+        "command": "mdb.exportToGo",
```

Comment:
You should also specify that this command is available only when a user is connected and for playground only and not other files opened in vscode.

Reply:
Done!